timescaledb/how-to-guides/connecting/dbeaver.md

```diff
@@ -1,15 +1,118 @@
+1.  Click the `Test Connection` button to check if the connection is successful.
+1.  Click the `Finish` button to connect to the database server. The server is
+    listed in the `Database Navigator` pane.
+
+    <img class="maincontent__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/timescale_cloud_dbeaver.png"alt="Connecting to Timescale Cloud using DBeaver"/>
+
+</procedure>
+
+### Connecting to Managed Service for TimescaleDB with DBeaver
+
+<highlight type="important">
+To ensure a secure network access to your Managed Service for TimescaleDB
+service, make sure you add the IP address of the machine running DBeaver to the
+`Allowed IP Addresses`. For more information about network security in Managed
+Service for TimescaleDB, see [the security section](https://docs.timescale.com/mst/latest/security/).
+</highlight>
+
+<procedure>
+
+1.  Sign in to the [Managed Service for TimescaleDB portal][mst-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to see the connection
+    information. Take a note of the `Host`, `Port`, `Database Name`, `User`, and
+    `Password`.
+1.  Start `DBeaver`.
+1.  In the `Database` menu, click `New Database Connection`.
+1.  In the `Connect to a database` window, search for `TimescaleDB`.
+1.  Select `TimescaleDB` and click `Next` to continue.
+1.  In the `Main` tab type the details for your connection, including the
+ `Host`, `Port`,`Database`, `Username` and `Password`. For more information
+ about these settings, see [the connection settings section][connection-settings].
+1.  Click the `Test Connection` button to check if the connection is successful.
+1.  Click the `Finish` button to connect to the database server. The server is
+    listed in the `Database Navigator` pane.
+
+    <img class="maincontent__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/mst_dbeaver.png"alt="Connecting to Managed Service for TimescaleDB using Dbeaver"/>
+
+</procedure>
+
+## Connection settings in `Connect to a database`
+
+This table provides the description and example values for the fields that
+appear in the `Main` tab of `Connection settings`.
+
+   |Setting|Example value|Description|
+   |-|-|-|
+   |Host|localhost, postgresql.example.com, <REMOTE_HOST>.tsdb.cloud.timescale.com, <REMOTE_HOST>.timescaledb.io|The fully qualified server name. The host name of your Timescale Cloud service or Managed Service for TimescaleDB service.|
+   |Port|5432|The port number of the TimescaleDB server, Timescale Cloud service or Managed Service for TimescaleDB service.|
+   |Username|postgres, tsdbadmin|The user name you want to log in with. Use `tsdbadmin` for Timescale Cloud service and Managed Service for TimescaleDB service|
+   |Password|*password*|The password for the account you are logging in with|
```

Comment:
```suggestion
   |Password|*password*|The password for the database service you are connecting to.|
```

Reply:
what I mean is eg in the case os TS Cloud, the account password is not the same as the service's password